bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/util/ThingHelper.java

```diff
@@ -81,7 +81,7 @@ public static boolean equals(Thing a, Thing b) {
     private static String toString(List<Channel> channels) {
         List<String> strings = new ArrayList<>(channels.size());
         for (Channel channel : channels) {
-            strings.add(channel.getUID().toString() + '#' + channel.getAcceptedItemType());
+            strings.add(channel.getUID().toString() + '#' + channel.getAcceptedItemType() + '#' + channel.getTriggerType());
```

Comment:
Ah, ok I missed that - I thought that is a general toString() method, which might also end up in log files or somewhere else. So all fine, thanks.